### ISSUE TITLE
Fix 'Missing form label' errors

### ISSUE THREAD

**Issue by JEThompson97** [UserOfficeProject/stfc-user-office-project#116]
Part of #68
> ### Missing form label
> **What it is:** A form control does not have a corresponding label.
> **Why it matters:** If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.

There are _several instances_ of this error throughout the app.
These include:

1.
![image](https://user-images.githubusercontent.com/31534888/117268292-08e7f200-ae4f-11eb-9842-1cceb9b69b98.png)

2.
![image](https://user-images.githubusercontent.com/31534888/117274844-5d8e6b80-ae55-11eb-979b-9c2bf63dbe62.png)
(same for all tabs: USER, REVIEWER, etc.)

/overwrite opened 2020-09-04

**Comment by JEThompson97** [comment]
There are a few instances where the WAVE tool _falsely claims_ accessibility issues with _material-ui components_.

[See this linked discussion](https://github.com/mui-org/material-ui/issues/23966) for more details

**Comment by JEThompson97** [comment]
It seems both instances of 'Missing form label' (screenshots above) may be fine to leave.

On further investigation, the textboxes in **screenshot 1** actually appear to have associated label elements (despite WAVE claiming them not to):
![image](https://user-images.githubusercontent.com/31534888/117278293-8a904d80-ae58-11eb-9655-3c325da7753b.png)

Which appears to be clear in the ARIA transcript:
![image](https://user-images.githubusercontent.com/31534888/117278366-9d0a8700-ae58-11eb-9e54-9797e300ecac.png)

And the textbox in **screenshot 2** is actually has `aria-hidden="true"` which would make it invisible to screen readers.

These findings imply this issue can be closed without changes.

**Comment by JEThompson97** [comment]
When I last worked on this, I found evidence to suggest these 'errors' were mistakes from the WAVE tool and don't need correction.

I think the best next step here would be to arrange a DR-like meeting to try to confirm OR disprove this hypothesis.

**Comment by JEThompson97** [comment]
Panda and I looked into this a bit more today.

We discovered when Material UI input components are given an `id`, these are automatically used to associate the label with the input. [Source with examples](https://material-ui.com/components/text-fields/)

This can be used to resolve 1. above. I will investigate whether this fix can be used to resolve 'missing form label' across all other instances.
